source/kubernetes.rst

```diff
@@ -80,7 +80,13 @@ Install Percona server for MongoDB on Kubernetes
 
    More details about secrets can be found in `Users <users.html>`_.
 
-6. After the operator is started, Percona Server for MongoDB cluster can
+6. The following command applies pre-generated certificates:
```

Comment:
user shouldn't use pre-generated certificates, it is highly unrecommended! we should add a link to TLS doc and explain that it is only for testing purposes he can use pre-generated certificates.